neuromation/cli/root.py

```diff
@@ -187,3 +195,20 @@ def _process_chunk(self, chunk: bytes, printable: bool) -> List[str]:
             for line in self._process_chunk(chunk, context.response_printable)
         ]
         self._print_debug(lines)
+
+    def _sanitize_header_value(self, text: str) -> str:
+        for token in self._find_all_tokens(text):
+            token_safe = self._sanitize_token(token)
+            text = text.replace(token, token_safe)
+        return text
+
+    def _sanitize_token(self, token: str, tail_len: int = 5) -> str:
+        if not (0 < tail_len < len(token) // 2):
```

Comment:
Why is `0` here? Do you suppose to support negative lengths?
I think the condition can be significantly simplified.

Reply:
`tail_len` is the parameter set up manually (currently, only in signature of `_sanitize_token`, in future it's possible to expose it further). Thus, we need to assert that it's a valid length. I agree it'll be more clear and idiomatic if the funciton fails on non-positive values of `tail_len`.

Reply:
Thanks for the reminder.
The idiomatic approach is don't add the unused functionality.
Drop `tail_len` from the method parameter, convert it into `tail_len = 5` local variable.